requirements.txt

```diff
@@ -6,3 +6,4 @@ pymongo==3.2.2
 pytz==2016.4
 requests[security]==2.10.0
 six==1.10.0
+girder-client==2.2.1
```

Comment:
this will get picked up in setup.py because of (regrettable) [magic](https://github.com/girder/girder_worker/blob/master/setup.py#L28-L71)